### DISCUSSION TITLE
Same git integration, multiple Vercel teams.

### DISCUSSION THREAD

**Discussion by dennispassway** [vercel/vercel#4555] [Help] [answered]
Hi!

TLDR: Is there a way to import from the same Github team into different Vercel teams?

## Situation
At our agency we use Vercel for hosting several clients. A client has 1, or more projects. The code for all of our clients is in GitHub in one team, the company's team. What I'm trying to do is create a Vercel team per client so that the hard limits only apply to that client. We would like to structure it this way so that a client is not affected by another client's actions (# of api builds, or waiting in a build queue on another client for example). Also, we can then see our billing per client.

## What I tried
In this example we have 2 Vercel teams. 'Team Internal' is our internal team, with internal projects. 'Team Client' is the client's team, with their projects. The Git integration is already set up with 'Team Internal', connected to our Github team. What I'm now trying to do is add that same github integration to 'Team Client'. When I try to do this I get the option to 'configure' the integration. That sends me to the settings page of Github, but I cannot find a way to get back to Vercel.

## What I expect
I expect to be able to add an existing git integration to multiple teams in Vercel. So in the example above the 'Team Internal' and 'Team Client' should both be able to connect to the same Github team, since the Vercel app is already installed there. I do not want to create a Github account per client but that seems to be the only solution at this moment.

So my question is: is there a way to import from the same Github team into different Vercel teams?

Thanks in advance.

**Comment by dennispassway** [comment, marked as answer]
I got an answer via support. For anyone running into this problem in the future:

"The short answer is that we currently don't have a way to allow for a GitHub organization to get connected to multiple Vercel teams. We are well aware of the flexible needs of agencies and are working hard to determine the best way to allow agencies to manage workflows for their clients. Keep in mind that resources like bandwidth are unlimited and this won't impact your clients. When it comes to more builds, have you ever run into any limitations up to this point? We do allow you to purchase more concurrent builds if needed."

**Comment by mylesthedev** [comment]
"We are well aware of the flexible needs of agencies and are working hard to determine the best way to allow agencies to manage workflows for their clients"

Two years later and no changes to this?

**Reply by leerob** [reply]
Could you expand more on what you're looking to accomplish?

**Reply by mylesthedev** [reply]
Basically we have 5-7 developers working on multiple Vercel hosted projects.

If we were to create a new Team for each project it would work out like so:
Team 1: $20USD x 7 = $140
Team 2: $20USD x 7 = $140
Team 3: $20USD x 7 = $140
Total Cost for 3 websites: $420USD per month.

Price is one factor but then there's also the nature of agencies e.g. developers come and go so adding and removing developers to multiple teams would become cumbersome.

We'd happily pay more for an agency account that allows us to use the platform in a more effective way :)
